### PR TITLE
Add Keystone 2468 Battery Holder 2xAAA

### PR DESCRIPTION
**Pull request by jbrgn**
https://www.digikey.com/product-detail/en/keystone-electronics/2468/36-2468-ND/303817

### CODE REVIEW

File: pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  (pad 61856d8f-3a7c-4219-b680-451922719511 (side tht) (shape round)
+   (position 6.35 20.32) (rotation 90.0) (size 3.6 1.8) (drill 1.02)
+  )
+  (polygon 326a1398-0a86-463f-9c7f-efb87600d382 (layer top_placement)
+   (width 0.2) (fill false) (grab_area true)
+   (vertex (position -12.395 26.59) (angle 0.0))
+   (vertex (position 12.395 26.59) (angle 0.0))
+   (vertex (position 12.395 -26.59) (angle 0.0))
+   (vertex (position -12.395 -26.59) (angle 0.0))
+   (vertex (position -12.395 26.59) (angle 0.0))
```
Comment (rnestler) [inline pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp:89]: The `12.395` and the `26.59` seem a bit odd. Shouldn't it be 12.4 and 26.6? From the datasheet at https://www.keyelco.com/userAssets/file/M65p28.pdf it looks like the device is 53.0 x 24.6mm. So 53.0 / 2 + 0.1 (half the trace width) = 26.6.

File: pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  )
+  (polygon d9eecf73-db75-4682-86b8-e9012ca4cc0f (layer top_documentation)
+   (width 0.2) (fill false) (grab_area true)
+   (vertex (position -12.195 26.39) (angle 0.0))
+   (vertex (position 12.195 26.39) (angle 0.0))
+   (vertex (position 12.195 -26.39) (angle 0.0))
+   (vertex (position -12.195 -26.39) (angle 0.0))
+   (vertex (position -12.195 26.39) (angle 0.0))
```
Comment (rnestler) [inline pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp:97]: Same here, but with 12.2 and 26.4 instead.

File: pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  )
+  (circle 345403ba-e446-49ff-851f-8e47619858b6 (layer top_placement)
+   (width 0.4) (fill false) (grab_area false) (diameter 2.8) (position 2.35 14.985)
+  )
+  (circle f6e38727-23d6-46f9-a00d-70c3b40b9da0 (layer top_placement)
+   (width 0.4) (fill false) (grab_area false) (diameter 2.8) (position -2.35 -14.985)
+  )
+  (circle 51f81c6a-8bff-4e25-99a1-5f7b9ad42d96 (layer top_placement)
+   (width 0.4) (fill false) (grab_area false) (diameter 2.8) (position 0.0 0.0)
+  )
```
Comment (rnestler) [inline pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp:122]: What are the circles for? Should they be holes for mechanical mounting? Or are they just helpers *if* the user want's to add them? But then they probably should be top documentation and not top placement, right?

**Review by rnestler — COMMENTED** [review]
Some comments, sorry it took so long until somebody took a look at it :slightly_smiling_face:

File: pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp (hunk 1 above)
Comment (dbrgn) [reply pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp:89]: The "Drawing" URL is more exact: http://www.keyelco.com/product-pdf.cfm?p=1033

52.98 / 2 + 0.1 = 26.59 :slightly_smiling_face:

File: pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp (hunk 2 above)
Comment (dbrgn) [reply pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp:97]: See above.

File: pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp (hunk 3 above)
Comment (dbrgn) [reply pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp:122]: We assume the holes are used to push out the batteries from the other side of the PCB.

I added two variants of the board. One with holes, the other without. In both cases, the hole size is shown on the documentation layer.

**Comment by dbrgn** [comment]
@rnestler @ubruhin updated!

**Review by ubruhin — APPROVED** [review]
Thanks!

**Review by dbrgn — APPROVED** [review]

**Pull request merged**